src/server/controllers/user/verify.js

```diff
@@ -1,10 +1,8 @@
-import { User } from '#schemas/users/userSchema.js';
+import { getUserByToken } from '#handlers/userHandlers.js';
 
 async function verify(req, res) {
   try {
-    const user = await User.findOne({
-      verificationToken: req.params.verificationToken,
-    });
+    const user = await getUserByToken(req.params.verificationToken);
```

Comment:
Can we desctructurizate tihs token?

We want to get clean props like so
f.ex
 ```
const { params.verificationToken = token } = req;
 ...
 getUserByToken(token); 
```

Reply:
We can.